src/test/java/bio/terra/service/filedata/DrsServiceTest.java

```diff
@@ -117,17 +129,9 @@ public class DrsServiceTest {
 
   private DRSPassportRequestModel drsPassportRequestModel;
 
-  private final AuthenticatedUserRequest authUser =
-      AuthenticatedUserRequest.builder()
-          .setSubjectId("DatasetUnit")
-          .setEmail("dataset@unit.com")
-          .setToken("token")
-          .build();
-
-  private String rasIssuer = "https://stsstg.nih.gov";
-
   @Before
   public void before() throws Exception {
+    drsIdService = new DrsIdService(appConfig);
```

Comment:
Just curious, is there a reason that we're not mocking `DrsIdService`?  Could we mock it then test it separately in its own test, rather than testing it implicitly in this test (not sure if that's the intention)?

Reply:
Sorry, I'm still not following!  I see that you mock the other inputs to `DrsService` so I'm not sure why not mocking this one makes a difference.

Reply:
Oh I see.  Yeah, I could mock as well. I just did it since I know that it only needs the hostname config value.  So it was a toss up between mocking the config the one call or just have a regular object with the one value.  Seemed like 6 of one, half dozen of the other.  I'll mock for consistency